### PR TITLE
Documentation for tool `tox`

### PR DESCRIPTION
**Pull request by Honny1**
#### Description:

This PR creates documentation for the use of `tox`. And disables the automatic execution of `flake8` when using `tox`.

### CODE REVIEW

File: docs/manual/developer/08_content_tests.md [hunk 1]
```diff
@@ -15,9 +15,44 @@ To add a script to be checked with mypy use the `mypy_test` macro in `tests/CMak
 
 ### Unit Tests
 
-`ctest` is used to run the unit tests for the `ssg` python package that is in the repo.
+`ctest` is used to run the unit tests for the `ssg` python package that is in the repo. But you need to build content and `ctest` isn't able to run unit tests with differed versions of python. To run only unit tests with differed version of python for the `ssg` python package is recommended to use `tox`.
```
Comment (jan-cerny) [inline docs/manual/developer/08_content_tests.md:18]: This is against the Style Guide https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer/04_style_guide.md#markup-languages which says that we shall have one sentence per line.

Comment (jan-cerny) [inline docs/manual/developer/08_content_tests.md:18]: Python should be with capital P.

Comment (jan-cerny) [inline docs/manual/developer/08_content_tests.md:18]: different

Comment (jan-cerny) [inline docs/manual/developer/08_content_tests.md:18]: I don't understand what you mean by "But you need to build content and ..." . If you mean that you need to build SCAP content in order to run unit tests then that's not true.

File: docs/manual/developer/08_content_tests.md [hunk 2]
```diff
@@ -15,9 +15,44 @@ To add a script to be checked with mypy use the `mypy_test` macro in `tests/CMak
...
+
+Note that `tox` creates a virtual environment that handles all dependencies defined in the test request file and performs unit tests with multiple versions of python. Thus, you can test your changes with different versions of python on your machine and don't have to wait for the upstream CI to check them for you.
```
Comment (jan-cerny) [inline docs/manual/developer/08_content_tests.md:20]: Python should be with capital P.

File: docs/manual/developer/08_content_tests.md [hunk 3]
```diff
@@ -15,9 +15,44 @@ To add a script to be checked with mypy use the `mypy_test` macro in `tests/CMak
...
+
+#### Execute Unit Tests via Tox
+
+Installation of `tox` and more advanced usage is described in [documentation](https://tox.wiki/en/4.11.3/).
+
+*You must be in the root directory of the project before running the `tox` command.*
+
+Runs tests with python2, python3.8, python3.9 and python3-latest on machine:
```
Comment (jan-cerny) [inline docs/manual/developer/08_content_tests.md:28]: Is the `python2` Python 2.7 or which version?

File: docs/manual/developer/08_content_tests.md (hunk 2 above)
Comment (jan-cerny) [inline docs/manual/developer/08_content_tests.md:20]: I would omit the "Note that"

**Review by jan-cerny — CHANGES_REQUESTED** [review]
This is a good idea and I love the various examples.

File: docs/manual/developer/08_content_tests.md (hunk 1 above)
Comment (Honny1) [reply docs/manual/developer/08_content_tests.md]: If I understand [documentation](https://complianceascode.readthedocs.io/en/latest/manual/developer/02_building_complianceascode.html#testing) correctly? It means that the unit tests are part of the `ctest` test suite, and before running `ctest` you need to execute the `./build_product` command to build the content.  For `tox`, just clone the repo, install `tox` and run it.

Comment (jan-cerny) [reply docs/manual/developer/08_content_tests.md]: But we're talking about Python unit tests of the `ssg` Python package. For them to run, you don't have to build anything, it's enough to just configure:

```
cd build 
rm -rf *
cmake ..
ctest -R python-unit-ssg-module
```

Comment (Honny1) [reply docs/manual/developer/08_content_tests.md]: Okay, I'll add it to the documentation. This information was missing from the documentation.

**Review by jan-cerny — APPROVED** [review]
I like these changes very much! They're really detailed and precise.

The TF CS9 fail is tracked in https://github.com/ComplianceAsCode/content/issues/11179 and isn't caused by the changes in this PR.

**Comment by Honny1** [comment]
@jan-cerny All tests were successful.

**Pull request merged**